### ISSUE TITLE
Suggestion: Folder icons in accent color

### ISSUE THREAD

**Issue by nclarius** [vinceliuice/Tela-icon-theme#175]
It would be very nice if there was a flavor of folder icons that automatically adapts to the accent color in KDE. The colors currently provided are a good start, but don't exactly match the predefined colors KDE offers, and with the next Plasma release it will even be possible to specify any custom accent color. There is also a [script](https://github.com/tantalising/kde-plasma-adaptive-wallpaper-accent-color) that automatically extracts a custom accent color from the wallpaper. Having visually consistent folder colors would be a pleasant addition.

[According to this](https://www.reddit.com/r/kde/comments/ri4zko/comment/howapa9/?utm_source=share&utm_medium=web2x&context=3), accent color can be achieved for window decorations with just two lines in the SVG changed; I imagine it would be possible in a similar way for icons?

Thank you for your work so far!

**Comment by MolassesLover** [comment]
Oh, woah! This is an awesome idea, thank you. I am going to implement this soon, just give me a little while.

**Comment by nclarius** [comment]
> Oh, woah! This is an awesome idea, thank you. I am going to implement this soon, just give me a little while.

That's cool; make sure to let me know if and when you get something!

**Comment by vinceliuice** [comment]
Now it's updated

**Comment by nclarius** [comment]
That was quick and it works flawlessly; thank you!

If I may make another suggestion: I turned out that the `Color-Scheme-Highlight` is a lighter shade than the accent color (I think it's what's used for selections) and the person figuring out the window decorations identified `Color-Scheme-ViewFocus` as the color matching the accent. Would you consider adopting that change or did you intentionally go with the less saturated variant?

**Comment by vinceliuice** [comment]
This follow the colorscheme

**Comment by nclarius** [comment]
Yes, but it's always a lighter variant than the main accent color.

**Comment by vinceliuice** [comment]
That's your colorscheme have a lighter color, default breeze icon is using Color-Scheme-Highlight for this, I think use the same as breeze is better because some color scheme's button focus color which is completely different from the highlight color.

**Comment by nclarius** [comment]
Hei again,

I just came across this: https://www.reddit.com/r/kde/comments/qwdszk/new_accent_colors_look_weird_with_dark_theme/htlzjaa/?context=3

The problem that icons can only access the `Highlight` color class, which was originally intended only as a selection background and is slightly tinted (lighter for light color schemes and darker for dark color schemes), has been fixed now, and it is now possible to access the actual accent color with `ColorScheme-ActiveText`. I just tried it out, and it looks a lot better, especially in the dark version which many people have (rightfully) complained about. I'm happy with my custom install, but you might want to consider adopting this change.

**Comment by nclarius** [comment]
For comparison:

with `ColorScheme-Highlight`:
![light-highlight](https://user-images.githubusercontent.com/13196860/150658891-c60c029b-8734-4de2-9c61-0fc53186e45d.png) ![dark-highlight](https://user-images.githubusercontent.com/13196860/150658865-a0ec6bf0-afc8-4490-951f-eb6bbdfaf668.png)

with `ColorScheme-ActiveText`:
![light-accent](https://user-images.githubusercontent.com/13196860/150658876-3801a96e-30c6-4a28-ad0a-6ffedaba4fea.png) ![dark-accent](https://user-images.githubusercontent.com/13196860/150658879-7d552c35-f77b-4e1d-84da-dab0b6acca2a.png)
